app/audit/models/ClaimARefundAuditModel.scala

```diff
@@ -38,18 +38,20 @@ case class ClaimARefundAuditModel(balanceDetails: Option[BalanceDetails],
   private def getAvailableCredit: Double = {
     balanceDetails.flatMap(_.availableCredit) match {
       case Some(_) => balanceDetails.get.availableCredit.get.toDouble.abs
-      case (_) => 0.00
+      case _ => 0.00
     }
   }
 
   private def getCreditType(credit: (DocumentDetailWithDueDate, FinancialDetail)): String = {
     val creditType: Option[CreditType] = credit._2.getCreditType
     val isPayment: Boolean = credit._1.documentDetail.paymentLot.isDefined
+    val taxYearString = s"${credit._1.documentDetail.taxYear - 1} to ${credit._1.documentDetail.taxYear} tax year"
```

Comment:
if you use case class Credit, then 'credit._1.documentDetail.taxYear - 1' can changed to credit.startYear val of def, which reads better